packages/govuk-frontend/src/govuk/components/exit-this-page/exit-this-page.yaml

```diff
@@ -2,11 +2,11 @@ params:
   - name: text
     type: string
     required: false
-    description: Text for the link. If `html` is provided, the `text` option will be ignored. Defaults to 'Exit this page'.
+    description: Text for the link. If `html` is provided, the `text` option will be ignored. Defaults to 'Emergency Exit this page' with 'Emergency' visually hidden.
```

Comment:
Should it be `"Emergency: Exit this page"`?

I can't help read "Emergency Exit…" as this sign

![Emergency exit sign](https://github.com/alphagov/govuk-frontend/assets/415517/f0ed445b-776d-4e21-8dcf-97d49ea32159)